seneca/seneca.d.ts

```diff
@@ -3,229 +3,227 @@
 // Definitions by: Peter Snider <https://github.com/psnider/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped/seneca
 
+declare function SNS(options?: SNS.Options): SNS.Instance;
```

Comment:
Fair enough.
I'll give that a try, re-test, and re-push.
I'll try to get this done today.